src/deadline_worker_agent/sessions/actions/run_attachment_download.py

```diff
@@ -212,15 +225,15 @@ def start(
         # returns root path to PathMappingRule mapping
         dynamic_mapping_rules: dict[str, PathMappingRule] = (
             session._asset_sync.generate_dynamic_path_mapping(
-                session_dir=session._session.working_directory,
+                session_dir=session.openjd_session.working_directory,
```

Comment:
Sure

Reply:
The intent was to to avoid exposing the entire `.openjd_session` property and expose only the subset of its API that we need to publicly.

Leaving it as a public attribute makes it more likely that external code does things that could conflict/interfere with the logic of the worker agent's outer `Session` class.